build.gradle

```diff
@@ -384,3 +389,14 @@ task allScaladoc(type: ScalaDoc) {
 
 // run by Heroku to build docs
 task stage(dependsOn: ':allScaladoc')
+
+import java.time.Instant
+import java.time.Duration
+
+gradle.taskGraph.beforeTask { task ->
```

Comment:
please remove `gradle.taskGraph.beforeTask` and `gradle.taskGraph. afterTask` - it makes the build very noisy.

Reply:
you can profile the build using the `--profile` or `--scan` commands with gradle.